website/docs/r/cdn_frontdoor_rule.html.markdown

```diff
@@ -0,0 +1,719 @@
+
+---
+
+A `url_rewrite_action` block supports the following:
+
+* `source_pattern` - (Required) Define the source pattern in the URL path to replace. Currently, source pattern uses a prefix-based match. To match all URL paths, use a forward slash (/) as the source pattern value.
+
+* `destination` - (Required) Define the destination path to use in the rewrite. The destination path overwrites the source pattern.
+
+* `preserve_unmatched_path` - (Optional) Append the remaining path after the source pattern to the new destination path? Possible values `true` or `false`. Defaults to `false`.
+
+---
+
+A `request_header_action` block supports the following:
+
+* `header_action` - (Required) The action to be taken on the specified `header_name`. Possible values inclued `Append`, `Overwrite` or `Delete`. `Append` the specified header gets added to the request with the specified value. If the header is already present, the value is appended to the existing header value using string concatenation. No delimiters are added. `Overwrite` action the specified header gets added to the request with the specified value. If the header is already present, the specified value overwrites the existing value. `Delete` action if the header specified in the rule is present, the header gets deleted from the request.
+
+* `header_name` - (Required) The name of the header to modify.
+
+* `value` - (Optional) The value to append or overwrite.
+
+~>**NOTE:** `value` is required if the `header_action` is set to `Append` or `Overwrite`.
+
+---
+
+* A `response_header_action` block supports the following:
+
+* `header_action` - (Required) The action to be taken on the specified `header_name`. Possible values inclued `Append`, `Overwrite` or `Delete`. `Append` the specified header gets added to the response with the specified value. If the header is already present, the value is appended to the existing header value using string concatenation. No delimiters are added. `Overwrite` the specified header gets added to the response with the specified value. If the header is already present, the specified value overwrites the existing value. `Delete` if the header specified in the rule is present, the header gets deleted from the response.
+
+* `header_name` - (Required) The name of the header to modify.
+
+* `value` - (Optional) The value to append or overwrite.
+
+~>**NOTE:** `value` is required if the `header_action` is set to `Append` or `Overwrite`.
+
+---
+
+A `parameter_name_override` block supports the following:
+
+* `param_name` - (Required) The name of the parameter.
+
+* `param_indicator` - (Required) Indicates the purpose of the parameter. Possible values include `Expires`, `KeyId` or `Signature`.
```

Comment:
```suggestion
* `parameter_name` - (Required) The name of the parameter.

* `indicator` - (Required) Indicates the purpose of the parameter. Possible values include `Expires`, `KeyId` or `Signature`.
```

Reply:
I removed this entire block as it is no longer valid since the secrets only support Managed and CMK... I think, I will have to look closer into why I added the block in the first place. 🤷‍♂️

Reply:
Oh... I remember now, this was part of the three legacy actions that are currently exposed but will be failed 90 days after GA. So I removed them from the code, but I forgot to remove part of one out of the documentation. Fixed.